packages/_examples/plugin/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
You'll want to exclude this from the lerna config so we don't accidentally publish this plugin (unless for some reason you want to)

Reply:
Yeah that is a good call out will do :)